src/java.base/share/classes/java/lang/invoke/MethodType.java

```diff
@@ -896,10 +906,6 @@ public boolean equals(Object x) {
         if (x instanceof MethodType mt) {
             return equals(mt);
         }
-        if (x instanceof ConcurrentWeakInternSet.WeakEntry<?> e
```

Comment:
The comment above about `WeakEntry` should also be removed.

Reply:
Will update